src/qml/bitcoin.cpp

```diff
@@ -184,6 +185,14 @@ int QmlGuiMain(int argc, char* argv[])
 
     engine.rootContext()->setContextProperty("nodeModel", &node_model);
 
+#ifdef __ANDROID__
+    AppMode app_mode(AppMode::MOBILE);
+#else
+    AppMode app_mode(AppMode::DESKTOP);
+#endif // __ANDROID__
+
+    qmlRegisterSingletonInstance<AppMode>("com.bitcoincore.qt", 1, 0, "AppMode", &app_mode);
```

Comment:
not 100% sure that is the namespace that should be used but it is what was defined in the AndroidManifest.xml also.  org.bitcoin.qt might make more sense for both.

Reply:
@hebasto what do you think about this URI namespace, see: https://doc.qt.io/qt-6/qqmlengine.html#qmlRegisterSingletonType